.pylintrc

```diff
@@ -16,14 +16,15 @@ persistent=yes
 # List of plugins (as comma separated values of python modules names) to load,
 # usually to register additional checkers.
 load-plugins=
+disable=multiple_types
```

Comment:
There's already a "disable" at line 37. And this specific one has no effect, as "multiple_types" is not a pylint message:

```
(DIRAC)  fstagni@DeskTof  ~/pyDevs/DIRAC   PilotSynchronizer  pylint --list-msgs | grep multiple_types
(DIRAC)  ✘ fstagni@DeskTof  ~/pyDevs/DIRAC   PilotSynchronizer  
```
What you probably look for is:
```
(DIRAC)  ✘ fstagni@DeskTof  ~/pyDevs/DIRAC   PilotSynchronizer  pylint --list-msgs | grep redefined-variable-type
:redefined-variable-type (R0204): *Redefinition of %s type from %s to %s*
```
which is already inserted in the integration branch (you can backport here if you want)

Reply:
I am a bit lost as this is the message pylint is asking me to include... and the fact is that pylint i uno longer giving me a warning... 
Conclusion: pylint is not really standard nor stable, right?

Reply:
Which version of pylint are you using? 
I use 
```
[macprophc] ~/cmtuser/Dirac_v0r0/DIRAC > python /Library/Python/2.7/site-packages/pylint/lint.py --version
lint.py 1.6.5,
astroid 1.4.9
Python 2.7.10 (default, Oct 23 2015, 19:19:21)
```

Reply:
Same as you:

```
DIRAC)  fstagni@DeskTof  ~/pyDevs/DIRAC   v6r19-fixes12  pylint --version                               
pylint 1.6.5, 
astroid 1.4.9
Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609]
```

Reply:
Can you issue:
```
pylint --list-msgs | grep multiple_types
```
and see if the message is defined?

Reply:
Fixed to redefined-variable-type